test/extended/OWNERS

```diff
@@ -9,6 +9,7 @@ reviewers:
   - squeed
   - dcbw
   - danwinship
+  - adambkaplan
```

Comment:
Need to be approver too (for extended build tests)?

Reply:
yeah you should be.